src/Microsoft.Health.Dicom.Core/Features/Common/RecyclableMemoryStreamOptions.cs

```diff
@@ -0,0 +1,38 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.IO;
+
+namespace Microsoft.Health.Dicom.Core.Features.Common;
+
+/// <summary>
+/// Represents the settings that may be configured for the shared instance of <see cref="RecyclableMemoryStreamManager"/>.
+/// </summary>
+public class RecyclableMemoryStreamOptions
```

Comment:
Are we planning to set this in the internal project?

Reply:
Yeah, I was planning to modify these options for PaaS over time (and for potential investigations). But this way we don't need further binary changes.